portfolio/src/main/webapp/js/custom-script.js

```diff
@@ -103,7 +102,20 @@ function populateDropdown(list, ID) {
   });
 }
 
+// inserts a functioning searchbar into the navigation bar of a page.
 function insertSearch() {
+  const searchVisual = createSearch();
+  const elementArray = Array.from(document.body.childNodes);
+  searchVisual.onkeyup = function() {
+    const resultArray = searchPages(elementArray);
+    console.log(resultArray);
```

Comment:
@wangtong-g comment:
https://github.com/kofimeighan/adelardos-capstone-2020/pull/17#discussion_r450558196
Is this for debugging or on purpose?

Reply:
Yes, it should not be there. Thank you for pointing that out.